docs/intro/index.md

```diff
@@ -1,6 +1,54 @@
 ---
-order: false
+order: 1
+description: Use commands to build, start, and add features to your first blockchain.
 parent:
   order: 1
   title: Introduction
----
\ No newline at end of file
+---
+
+# Quickstart
+
+Use these three commands to build, start, and add features to your first blockchain.
+
+## Create a blockchain
+
+To create a blockchain:
+
+```
+starport app github.com/username/myapp && cd myapp
+```
+
+The `app` command creates the blockchain directory `myapp` and scaffolds a [Cosmos SDK](https://docs.cosmos.network/) blockchain.
+
+## Run a blockchain
+
+To run a blockchain in your development environment:
+
+```
+starport serve
+```
+
+The `serve` command installs dependencies, builds, initializes, and starts the blockchain.
```

Comment:
@fadeev is this description of `starport serve` still accurate? what changes for v0.16?

Reply:
Thanks! Updated the docs.